build/Dockerfile

```diff
@@ -1,16 +0,0 @@
-FROM registry.access.redhat.com/ubi7/ubi-minimal:latest
```

Comment:
I think we should allow the user to deploy the operator as the dockerfile.rhel7 is for internal build